packages/markdown-docx/package.json

```diff
@@ -26,8 +26,8 @@
     "lint": "eslint .",
     "postlint": "npm run licchk",
     "licchk": "license-check",
-    "test": "jest --timeOut=10000",
-    "test:cov": "npm run lint && jest --timeOut=10000 --coverage",
+    "test": "mocha -t 10000",
+    "test:cov": "npm run lint && nyc --reporter=text npm run test",
```

Comment:
What is the use of `--reporter=text` and why is it necessary?

Reply:
Not knowing the exact use but it gives the coverage in form of table which I think was quite awesome so used it.

Reply:
`reporter=text` is the default configuration of `nyc`. You might not need it.

Reply:
Just read it. The command `--reporter=text` gives a text summary coverage report.

Reply:
OK I will remove the `--reporter=text` from the script.